src/Python/setup-regularisers.py.in

```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/env python
 
-from setuptools import Extension, setup, find_packages
+from setuptools import Extension, setup
 from Cython.Distutils import build_ext
```

Comment:
I think we need to use `Cython.Build.cythonize` instead à la https://docs.cython.org/en/latest/src/userguide/source_files_and_compilation.html#configuring-the-c-build